src/components/Item/ViewItem.tsx

```diff
@@ -0,0 +1,111 @@
+import { useParams } from "react-router-dom";
+import { Item, fetchItem } from "../../api/service/item.service";
+import ItemInventoryTable from "../ItemExploration/ItemInventoryTable";
+import ItemEntry from "../ItemEntry/ItemEntry";
+
+function ManualItemEntry(props: { upc: string }) {
+    return (
+        <>
+            <Alert variant="filled" severity="info">
+                No item found with UPC {props.upc}. Please enter this item's information below.
+            </Alert>
+            <Box sx={{marginTop: 2}}>
+                <ItemEntry upc={props.upc}></ItemEntry>
+            </Box>
+        </>
+    )
+}
+
+function ItemInfo(props: { item: Item }) {
+    const visibleFields = [
+        { label: "UPC", key: "upc" },
+        { label: "Name", key: "name" },
+        { label: "Brand", key: "brand" },
+        { label: "Manufacturer", key: "manufacturer" },
+        { label: "Presentation", key: "presentation" },
+        { label: "Tablets", key: "tablets" },
+        { label: "Dosage", key: "dose" },
+        { label: "Category", key: "category" }
+    ];
+
+    return (
+        <>
+            <Paper elevation={3} variant="elevation">
+                <Box sx={{ padding: 5 }}>
+                    <Typography variant="h5">
+                        {props.item.name}
+                        <IconButton component={Link} href={"/intake-inventory?upc="+props.item.upc}><ModeEdit/></IconButton>
+                    </Typography>
+                    <Table size="small">
+                        <TableBody>
+                            {visibleFields.map((visibleField, index) => (
+                                <TableRow key={index}>
+                                    <TableCell sx={{fontWeight: "bold"}}>{visibleField.label}</TableCell><TableCell>{props.item[visibleField.key as keyof Item]}</TableCell>
+                                </TableRow>
+                            ))}
+                        </TableBody>
+                    </Table>
+                    <Box sx={{ marginTop: 5 }}>
+                        <ItemInventoryTable upc={props.item.upc}></ItemInventoryTable>
+                    </Box>
+                </Box>
+            </Paper>
+        </>
+    );
+}
+
+function ViewItem() {
+    const { upc } = useParams();
+    const [item, setItem] = useState<Item | null>(null);
+    const [itemLoaded, setItemLoaded] = useState<boolean>(false);
+
+    async function getItem() {
+        if (upc != undefined) {
+            const { data } = await fetchItem(upc);
+
+            if (data.item != null) {
+                setItem(data.item as Item);
+            }
+            setItemLoaded(true);
+        }
+    }
+
+    useEffect(() => {
+        getItem();
+    })
+
+    if (upc == undefined) {
+        return (
+            <Alert variant="filled" severity="warning">
+                No UPC provided.
+            </Alert>
+        );
+    }
+
+    if (itemLoaded) {
+        if (item == null) {
```

Comment:
```suggestion
        if (!item) {
```

null is a falsey value.